components/event-publisher-proxy/Makefile

```diff
@@ -26,8 +26,11 @@ dep-status-local:
 mod-verify-local:
 	GO111MODULE=on go mod verify
 
+# Use it before commit
+check-code: check-imports fmt-local vet-local lint
+
 test-local:
-	GO111MODULE=on go test ./...
+	GO111MODULE=on go test ./... -coverprofile cover.out
```

Comment:
do we really need to always write the coverage to a file? Usually developers use an IDE to check the the coverage and these then also take care of generating the coverage for you

Reply:
Added only to have the same coverage behavior as in "eventing-controller"